Examples/green_button_data_example/parse_and_run_green_button.ipynb

```diff
@@ -0,0 +1,299 @@
+    "kW_data = df_23.loc['2023-01-01 07:00:00':'2024-01-01 06:59:00']['Wh'] / 250.0\n",
+    "print(len(kW_data))\n",
+    "kW_data.to_csv('15_min_load.csv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "4534f881",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Optional: example conversion from 15 minute data in Wh to hourly data in kW\n",
+    "\n",
+    "hourly_data = [0] * 8760\n",
+    "for i, v in enumerate(df_23.loc['2023-01-01 07:00:00':'2024-01-01 06:59:00']['Wh'].values):\n",
+    "    hourly_data[int(i / 4)] += v / 1000.0\n",
+    "\n",
+    "idx = range(0, 8760)\n",
+    "df_hrly = pd.DataFrame.from_dict({'time' : idx, 'kW' : hourly_data})\n",
+    "df_hrly.to_csv('hourly_load.csv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "789bb8b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download rate from URDB and save as file. If rate has already been downloaded, use file\n",
+    "def get_urdb_rate_data(page, key):\n",
+    "\n",
+    "    # Full API can be viewed at: https://openei.org/services/doc/rest/util_rates/?version=8\n",
+    "    urdb_url = 'https://api.openei.org/utility_rates?format=json&detail=full&version=8'\n",
+    "    get_url = urdb_url + '&api_key={api_key}&getpage={page_id}'.format(api_key=key, page_id=page)\n",
+    "    print(get_url)\n",
+    "\n",
+    "    filename = \"urdb_rate_{}.json\".format(page)\n",
+    "    print(filename)\n",
+    "\n",
+    "    if not os.path.isfile(filename):\n",
+    "        print(get_url)\n",
+    "        resp = requests.get(get_url, verify=certifi.where())\n",
+    "        data = resp.text\n",
+    "        # Cache rate as file\n",
+    "        if \"error\" not in data:\n",
+    "            with open(filename, 'w') as f:\n",
+    "                f.write(json.dumps(data, sort_keys=True, indent=2, separators=(',', ': ')))\n",
+    "    else:\n",
+    "        with open(filename, 'r') as f:\n",
+    "            data = json.load(f)\n",
+    "\n",
+    "    return data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0d90f6c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Starting data download for solar using 1 thread workers.\n",
+      "File already exists. Skipping download: .\\nsrdb_39.74_-105.17_nsrdb-GOES-conus-v4-0-0_15_2023.csv\n",
+      "https://api.openei.org/utility_rates?format=json&detail=full&version=8&api_key=RkdVzEGa7mlEGqV9jzwjhrh95ACZOr6PFr0K0yTc&getpage=67a65130767b93f0b5044b9a\n",
+      "urdb_rate_67a65130767b93f0b5044b9a.json\n",
+      "Year one savings  876.0264276434835\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Download 15 minute data from NSRDB\n",
+    "building_path = \".\"\n",
+    "year = 2023\n",
+    "\n",
+    "sam_api_key = \"YOUR_API_KEY\"\n",
+    "sam_email = \"YOUR_EMAIL\"\n",
```

Comment:
move these key variables to the very top so it's easy to set from the beginning. also, name them nrel_developer_api_key and nrel_developer_email so it's clear it's not SAM-based